packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -241,6 +256,7 @@ export const MenuItem = forwardRef<HTMLLIElement, MenuItemProps>(
         aria-disabled={isDisabled ?? undefined}
         aria-haspopup={hasChildren ?? undefined}
         aria-expanded={hasChildren ? submenuOpen : undefined}
+        aria-label={ariaLabel || label}
```

Comment:
@Kilian-Collender I'm assuming this why the `avt` CI check is failing. What was the reasoning for adding this one?

Reply:
I can remove it again, but I had do it to make it easier to label and access the menuItem in the tests. I'm not 100% sure why it is complaining in the AVT as the visible label should match or include the text